providers/infrastructure-aws/v2.0.0-beta.1/cconly/base.yaml

```diff
@@ -142,24 +142,27 @@ spec:
                 type: string
               node:
                 type: string
-          proxy:
```

Comment:
I would bump the version number on the annotation at line 12 of this file `identifier.cc.vmware.com` and update Shimon on the UX team that the format of the variables has changed. We need to do the same for each clusterclass.

Reply:
Got it. tahnks